server/stats/event_service.py

```diff
@@ -60,7 +60,11 @@ def __init__(self, api_accessor: ApiAccessor):
         Else, returns None
         """
         self._logger.debug("Recording %d events", len(queue))
-        response, content = await self.api_accessor.update_events(queue, player_id)
+        try:
+            response, content = await self.api_accessor.update_events(queue, player_id)
+        except ConnectionError:
+            self._logger.error("Failed to update events: connection error")
+            return None
```

Comment:
Let's catch exception with some info and raise exception with no info.

Reply:
Error type was logged further down the stack. Besides this function already returned None in some cases where things went wrong - do you want to refactor all its callers, too?